sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -895,16 +895,10 @@ case class ShowTablesCommand(
  *   SHOW TBLPROPERTIES table_name[('propertyKey')];
  * }}}
  */
-case class ShowTablePropertiesCommand(table: TableIdentifier, propertyKey: Option[String])
-  extends RunnableCommand {
-
-  override val output: Seq[Attribute] = {
-    val schema = AttributeReference("value", StringType, nullable = false)() :: Nil
-    propertyKey match {
-      case None => AttributeReference("key", StringType, nullable = false)() :: schema
-      case _ => schema
-    }
-  }
+case class ShowTablePropertiesCommand(
+    table: TableIdentifier,
+    propertyKey: Option[String],
+    override val output: Seq[Attribute]) extends RunnableCommand {
```

Comment:
do we need the `override val`?

Reply:
![image](https://user-images.githubusercontent.com/46485123/106468154-87439980-64d8-11eb-89cb-c04c8316d323.png)